-------------------------------------------------------------------------------------------------------------

异常时间：2022-07-14 01:16:06,179 
异常级别：ERROR
异常内容：给定的端口名称未以 COM/com 开头或未解析为有效的串行端口。
参数名: portName
System.ArgumentException: 给定的端口名称未以 COM/com 开头或未解析为有效的串行端口。
参数名: portName
   在 System.IO.Ports.SerialStream..ctor(String portName, Int32 baudRate, Parity parity, Int32 dataBits, StopBits stopBits, Int32 readTimeout, Int32 writeTimeout, Handshake handshake, Boolean dtrEnable, Boolean rtsEnable, Boolean discardNull, Byte parityReplace)
   在 System.IO.Ports.SerialPort.Open()
   在 Utils.AutoSerialPort.OpenPort() 位置 E:\创维MES项目\程序\数据采集\生产数据采集程序\MDC\Utils\Common\AutoSerialPort.cs:行号 395
-------------------------------------------------------------------------------------------------------------

异常时间：2022-07-14 03:28:29,090 
异常级别：ERROR
异常内容：rowKey左前缀查询失败：无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。
System.IO.IOException: 无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。 ---> System.Net.Sockets.SocketException: 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   在 System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   在 Thrift.Transport.TStreamTransport.Read(Byte[] buf, Int32 off, Int32 len)
   在 Thrift.Transport.TTransport.ReadAll(Byte[] buf, Int32 off, Int32 len)
   在 Thrift.Protocol.TBinaryProtocol.ReadAll(Byte[] buf, Int32 off, Int32 len)
   在 Thrift.Protocol.TBinaryProtocol.ReadI32()
   在 Thrift.Protocol.TBinaryProtocol.ReadMessageBegin()
   在 Hbase.Client.recv_scannerOpenWithPrefix() 位置 E:\创维MES项目\程序\数据采集\生产数据采集程序\MDC\Utils\HBaseClass\Hbase.cs:行号 3718
   在 Hbase.Client.scannerOpenWithPrefix(Byte[] tableName, Byte[] startAndPrefix, List`1 columns, Dictionary`2 attributes) 位置 E:\创维MES项目\程序\数据采集\生产数据采集程序\MDC\Utils\HBaseClass\Hbase.cs:行号 3687
   在 Utils.HBaseClass.GetDataFromHBase.GetRowsWithPrefix(String _rowKey, String table) 位置 E:\创维MES项目\程序\数据采集\生产数据采集程序\MDC\Utils\HBaseClass\GetDataFromHBase.cs:行号 304
-------------------------------------------------------------------------------------------------------------

异常时间：2022-07-14 09:00:55,834 
异常级别：ERROR
异常内容：帐号登录状态更新失败在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: SQL Network Interfaces, error: 26 - 定位指定的服务器/实例时出错)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 YJ.Data.SqlServerProvider.PrepareCommand(SqlCommand cmd, SqlConnection conn, SqlTransaction trans, CommandType cmdType, String cmdText, DbParameter[] cmdParms)
   在 YJ.Data.SqlServerProvider.ExecuteAction(String sql, DbParameter[] par, DbTransaction trans, String DBName, Int32 time)
   在 YJ.Data.SqlServerProvider.ExecuteAction(String sql, String DBName)
   在 MDC.BaseUI.ChangeLoginState(String account, String IP) 位置 E:\创维MES项目\程序\数据采集\生产数据采集程序\MDC\MDC\BaseUI.cs:行号 429
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:-1,State:0,Class:20
